src/php/bin/php_extension_doxygen_filter.awk

```diff
@@ -12,38 +12,63 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function sed_gensub(regexp, replacement, how, target,     cmd_, ret_) {
+    if (!target) {
+        target = $0
+    }
+    gsub(/'/, "'\"'\"'", target);
+    gsub(/\\\\/, "\\", regexp);
+
+    #cmd_ = "printf '" target "' | sed -E '/" regexp "/!{q1};{s//" replacement "/" tolower(how) "}'";
+    cmd_ = "printf '" target "' | sed -E 's/" regexp "/" replacement "/" tolower(how) "'";
+    if (cmd_ | getline ret_ != 1) {
+        close(cmd_);
+        error = "ERROR: running command: " cmd_ ", ret_: " ret_;
+        exit;
+    }
+    close(cmd_);
+    return ret_;
+}
+
 BEGIN {
     namespace = "Grpc";
     className = "";
     classDocComment = "";
-    delete methods; # methods[method][doc|args|static]
-    delete constants; # constants[i][name|doc]
+
+    delete methodNames; # methodNames[i]
+    delete methodArgs; # methodArgs[methodName]
+    delete methodDocs; # methodDocs[methodName]
+    delete methodStatics; # methodStatics[methodName], 1 if static
```

Comment:
Perhaps consider commenting it this way `# methodName => 1 if static`.

Reply:
addressed